tex/measure/lebesgue-int.tex

```diff
@@ -80,10 +87,25 @@ \section{The definition}
 	The constant function $1$ is \emph{not} integrable on $\RR$,
 	since $\int_\RR 1 \; d\mu = \mu(\RR) = +\infty$.
 \end{example}
-For this reason, I will usually prefer the term ``integrable''.
+For this reason, I will usually prefer the term ``absolutely integrable''.
```

Comment:
surely this is what you mean right (you've just complained ``integrable'' is confusing terminology above)

Reply:
yep!